src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -41,15 +42,17 @@
     private $resourceClassResolver;
     private $operationMethodResolver;
     private $operationPathResolver;
+    private $requestContext;
 
-    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver, OperationPathResolverInterface $operationPathResolver)
+    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver, OperationPathResolverInterface $operationPathResolver, RequestContext $requestContext)
```

Comment:
if you prefer, we can provide easily a bc layer by instantiating the request context with its default values.